### PR TITLE
formatting AD content

### PR DESCRIPTION
**Pull request by MBishop17**
related to #227.

### CODE REVIEW

File: content/rancher/v2.x/en/tasks/global-configuration/authentication/_index.md [hunk 1]
```diff
@@ -12,65 +12,55 @@ You have three options for user authentication in {{< product >}}:
...
+    <br/>
+	<br/>
+  	> **Note:** This field is only for search bases and not for search filters.
+	>
+	>- If your users and groups are in the search base, complete only the User Search Base.
+	>- If your groups are in a different search base, you can optionally complete the Group Search Base. This field is dedicated to searching groups, but is not required.
+
+5.	If your Active Directory deviates from the standard AD schema, complete the **Customize Schema** form to match it. Otherwise, skip this step.
+
+	>**Search Attribute** By default, the Search Attribute field defaults with three specific values: `sAMAccountName|sn|givenName`. After AD is configured, when a user enters text to add users or groups, Rancher automatically queries the AD server and attempts to match fields by sAMAccountName, last name, or first name. Rancher specifically searches for users/groups that begin with the text entered in the search field.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/global-configuration/authentication/_index.md]: we should keep the fact that this was only available as of v2.0.1

**Comment by deniseschannon** [comment]
One small nit on making sure we add in the version.

**Comment by MBishop17** [comment]
Added the version back in. Also, I made a file for each auth provider.

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**